probes/hasGitHubWorkflowPermissionUnknown/def.yml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2024 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: hasGitHubWorkflowPermissionUnknown
```

Comment:
name should be inverted?

Reply:
https://github.com/ossf/scorecard/pull/3816/commits/0a0ba747b9c7a8470c52beeb97fb9ecd68166fd7